.github/workflows/instance-tests.yaml

```diff
@@ -83,7 +83,7 @@ jobs:
           make instance-standard-test
 
       - name: Slack Notification
-        if: ${{ failure() || cancelled() }}
+        if: ${{ github.event_name != 'push' && (failure() || cancelled()) }}
```

Comment:
Hmm... would we want to notify on a push to `main`? I think we might be wanting to exclude non-main branches here. Unless there's context I'm missing?

Reply:
@cmars , I have edited the condition. I hope this match pushes to `main`
Thanks :)